### PR TITLE
8 Bugs and Errors

### PR DESCRIPTION
**Pull request by magdalena-u**
chapter 8: Bugs and Errors:
8_1 Retry
8_2 The locked box

### CODE REVIEW

File: 8_1_Retry.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return a * b;
+    } else {
+        throw new MultiplicatorUnitFailure('Klunk');
+    }
+}
+
+//Solution
+
+function reliableMultiply(a, b) {
+    for (;;) {
```
Comment (michal-lepik) [inline 8_1_Retry.js]: Try using recursion instead of a loop :)

File: 8_2_locked.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    _content: [],
+    get content() {
+        if (this.locked) throw new Error('Locked!');
+        return this._content;
+    },
+};
+
+//Solution
+
+function withBoxUnlocked(value) {
```
Comment (grzegorz-bielski) [inline 8_2_locked.js]: `value` is misleading, name it `callback` / `func`

File: 8_2_locked.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    } catch (er) {
+        throw er;
+    } finally {
+        box.lock();
+    }
+}
+
+//Test
+
+withBoxUnlocked(function() {
```
Comment (grzegorz-bielski) [inline 8_2_locked.js:36]: Arrow function would look better in here

File: index.html [hunk 4]
```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <script defer src="fizzbuzz.js"></script>
+        <script defer src="8_1_Retry.js"></script>
```
Comment (grzegorz-bielski) [inline index.html:7]: you could use `<script type="module" src="/index.js"/>`  that imports `index.js` which is exporting other modules

**Review by michal-karbasz — APPROVED** [review]

**Pull request merged**